### ISSUE TITLE
Position of the track infos

### ISSUE THREAD

**Issue by stefaa97** [musicbeeremote/mbrc#242]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Hi, I would like to see a fix of this problem:

Before:
![image](https://user-images.githubusercontent.com/116920204/198855191-d70db87b-1364-4e0c-8f6a-381e4e15bead.png)

After closed e reopened:
![image](https://user-images.githubusercontent.com/116920204/198855219-bce675c2-c8a6-4f86-93a1-cf9c2535fec5.png)

Seems like the position of the text not change dinamically

**Comment by kelsos** [comment]
Hey, @stefaa97 any more information about the device and Android version?

**Comment by stefaa97** [comment]
> Hey, @stefaa97 any more information about the device and Android version?

Yeah, I have tried on Android TV and Android 12. It happens with orizontal orientation if the previous track had a shorter name
